airmozilla/popcorn/views.py

```diff
@@ -92,7 +139,8 @@ def get(self, request, slug, form=None, conflict_errors=None):
             return self.cant_edit_event(event, request)
 
         context = {
-            'event': event
+            'event': event,
+            'slug': slug,
```

Comment:
Seems strange but if you say so, let's leave it then. 
"Hey Mike, what's your first name? Mike? Oh cool. Thanks Mike for informing me"  :)